server/src/main/java/com/objectcomputing/checkins/util/googleapiaccess/GoogleAccessor.java

```diff
@@ -45,7 +46,7 @@ public GoogleAccessor(@Property(name = "check-ins.application.name") String appl
      */
     public Drive accessGoogleDrive() throws IOException {
 
-        String apiScope = environment.getProperty("check-ins.application.scopes.scopeForDriveApi", String.class).orElse("");
+        String apiScope = environment.getProperty("check-ins.application.google-api.scopes.scopeForDriveApi", String.class).orElse("");
```

Comment:
Good catch. I must have broken this when I updated scope config as part of directory api story.